sample-docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  lnd:
+    container_name: lnd
+    image: docker.io/lightninglabs/lnd:v0.18.4-beta
+    restart: unless-stopped
+    stop_grace_period: 5m30s
+    volumes:
+      - ~/lnd:/root/.lnd
+    ports:
+      - "9735:9735"
+      - "10009:10009"
+    command:
+        lnd
```

Comment:
Here we should add feature bits advertising for onion messages and allowing messages to lndk